pkg/storage/stores/shipper/downloads/table_manager.go

```diff
@@ -163,10 +163,20 @@ func (tm *TableManager) getOrCreateTable(spanCtx context.Context, tableName stri
 	return table
 }
 
-func (tm *TableManager) syncTables(ctx context.Context) error {
+func (tm *TableManager) syncTables(ctx context.Context) (err error) {
```

Comment:
I prefer the other style of not defining the variable name in the method.

This is usually how we do it and I like that `return nil` at the end is a little more explicit.